src/logion/services/idenfy/idenfy.service.ts

```diff
@@ -149,13 +153,13 @@ export class EnabledIdenfyService extends IdenfyService {
         await this.locRequestService.update(json.clientId, async request => {
             request.updateIdenfyVerification(json, raw.toString());
             for(const file of files) {
-                request.addFile(file);
+                request.addFile(file, true);
```

Comment:
As the LOC request is still draft, the item should not be considered as already reviewed and approved.